circuits/query/query.circom

```diff
@@ -0,0 +1,65 @@
+pragma circom 2.0.0;
+include "../../node_modules/circomlib/circuits/mux3.circom";
+include "../../node_modules/circomlib/circuits/bitify.circom";
+include "../../node_modules/circomlib/circuits/comparators.circom";
+include "comparators.circom";
+
+/*
+  Operators:
+ "0" - equals
+ "1" - less-than
+ "2" - greater-than
+ "3" - in
+ "4" - notin
+*/
+template Query (valueLevels) {
```

Comment:
Maybe `valueLenght` name for the parameter would be more accurate, because `levels` makes sense in context of Merkle Trees, but not arrays.

Reply:
renamed